libkbfs/block_tree.go

```diff
@@ -496,6 +552,9 @@ func (bt *blockTree) getBlocksForOffsetRange(ctx context.Context,
 				res.nextBlockOffset.Less(minNextBlockOffset)) {
 			minNextBlockOffset = res.nextBlockOffset
 		}
+		if res.firstBlock {
+			gotFirstBlock = true
```

Comment:
Ooh, I didn't realize that indicated the end of a file. I added more logic to prevent us from moving on if the first block gets `context.DeadlineExceeded`.